atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/schema/KeyValueServiceMigrator.java

```diff
@@ -206,6 +206,8 @@ private void migrateTables(Set<TableReference> tables,
                                long migrationTimestamp,
                                ExecutorService executor,
                                GeneralTaskCheckpointer checkpointer) {
+        processMessage("Migrating tables at migrationTimestamp " + migrationTimestamp,
```

Comment:
It sends the message to an `OutputPrinter`. There's a chain of `processMessage` methods - they could all be `printMessage`?

I suspected that we'd break `__LARGE_INTERNAL_PRODUCT` if we renamed them all, but turns out said behemoth does not call the method directly.